fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckBottomSheet.kt

```diff
@@ -54,6 +54,9 @@ fun ReviewQualityCheckBottomSheet(
                     onProductRecommendationsEnabledStateChange = {
                         store.dispatch(ReviewQualityCheckAction.ToggleProductRecommendation)
                     },
+                    onReviewGradeLearnMoreClick = {
+                        // Bug TODO
```

Comment:
Just created [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1847740) to handle this.

Reply:
thanks! I'll plug in the bug number